src/main/java/blackjack/domain/card/Hand.java

```diff
@@ -21,16 +21,18 @@ public void add(Card card) {
     }
 
     public int calculate() {
-        return sumScore().getScore();
+        Score score = sumScore();
+        return score.getScore();
     }
 
     public boolean isBusted() {
         Score score = sumScore();
         return score.isBusted();
     }
 
-    public BlackJackResult match(Hand hand) {
-        return sumScore().match(hand.sumScore());
+    public boolean isBlackJack() {
+        Score score = sumScore();
+        return score.isMaxValue() && hand.size() == BLACKJACK_CARD_SIZE;
```

Comment:
1. 블랙잭 여부에 대한 판단을 Hand에서 처리하는게 맞나요? Hand 객체 역할은 무엇인가요?
2. Score와 HandCalculator 객체는 각각 맡은 역할이 무엇인가요? 21이란 값이 왜 두 클래스에 중복으로 생겨났을까요?

Reply:
블랙잭의 기준은 21이 되면서 2장의 카드만 가져야 하기 때문에
카드의 합을 구할 수 있으면서 몇 장의 카드를 가지고 있는지에 대한 정보도 가지고 있는 hand의 역할이어도 괜찮겠다 생각했습니다.

그렇지 않으면 gamer에서 hand를 꺼내서 해당 로직을 처리해야 하는데 그것보다는 hand객체에게 그 역할을 위임하는 것이 맞겠다 싶었는데 hand를 꺼내서 확인을 해보는게 더 나은 방법일까요?

Reply:
게임 규칙을 책임지고 관리하는 객체가 Score로 보이는데 hand에서도 따로 처리하는 로직이 있어서 물어본거였어요. 책임이 분산되는 문제가 생기니깐요.
이렇게 수정하면 score를 통해 게임 결과를 알 수 있지 않을까요?
```suggestion
        return score.isBlackJack(hand.size());
```

Reply:
아 이렇게 되면 블랙잭이 되기 위한 카드 사이즈인 2도 Score에서 관리할 수 있겠네요!